VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionResponsesFragment.java

```diff
@@ -119,6 +119,23 @@ public void onCreate(Bundle savedInstanceState) {
     }
 
     @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        switch (discussionThread.getType()) {
+            case DISCUSSION:
+                getActivity().setTitle(R.string.discussion_title);
+                break;
+            case QUESTION:
+                if (discussionThread.isHasEndorsed()) {
+                    getActivity().setTitle(R.string.course_discussion_answered_title);
+                } else {
+                    getActivity().setTitle(R.string.course_discussion_unanswered_title);
```

Comment:
This could be shortened to use the conditional `?:` operator I guess. Up to you though.